crates/bevy_render/src/renderer/render_device.rs

```diff
@@ -64,11 +64,17 @@ impl RenderDevice {
                         })
                 }
             }
-            _ => self.device.create_shader_module(desc),
+            _ => unsafe {
```

Comment:
SAFETY comment here please! Or at least a comment explaining why we're disabling these checks.

Reply:
```suggestion
            // SAFETY: we are interfacing with shader code, which may contain undefined behavior,
            // such as indexing out of bounds.
            // The checks required are prohibitively expensive and a poor default for game engines.
            // TODO: split this method into safe and unsafe variants, and propagate the safety requirements from
            // https://docs.rs/wgpu/latest/wgpu/struct.Device.html#method.create_shader_module_trusted to the unsafe form.
            _ => unsafe {
```